components/BackButton/index.tsx

```diff
@@ -0,0 +1,13 @@
+'use client';
+
+import { useRouter } from 'next/navigation';
+
+export default function BackButton() {
+    const router = useRouter();
+
+    return (
+        <button className="back-link-button" onClick={() => router.back()}>
+            &larr; Back
```

Comment:
L10: Just used the arrow element until we have the Google Material symbols available.